deploy/helpers/mockPool.ts

```diff
@@ -0,0 +1,64 @@
+import { BigNumber } from "ethers";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+async function deployMockCurvePool(
+  env: HardhatRuntimeEnvironment,
+  name: string,
+  underlying0: string,
+  otherUnderlyings: string[]
+) {
+  const { deployer } = await env.getNamedAccounts();
+  const { deploy, execute, getOrNull, read, get } = env.deployments;
+
+  const underlying = await get(underlying0);
+
+  const underlyingDecimals = await read(
+    underlying0,
+    { from: deployer },
+    "decimals"
+  );
+
+  const isDeployed = await getOrNull(name);
+
+  if (!isDeployed) {
+    await deploy(name, { contract: "MockCurve", from: deployer, args: [] });
+
+    await execute(name, { from: deployer }, "addToken", 0, underlying.address);
+
+    let i = 1;
```

Comment:
I guess is is just the index of `otherUnderlyings` that corresponds to `tokenName` right?

Reply:
yeah I think so

to be fair I just pulled this in from main :joy: but looks like it https://github.com/lindy-labs/sc_solidity-contracts/blob/gp-connect-contracts/contracts/mock/MockCurvePool.sol#L17

Reply:
I guess this could be written inside the for loop right?